packages/cli/src/Db.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-cycle */
```

Comment:
[This rule](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md) treads the line between style and architecture/organization. I agree with keeping it to avoid circular dependencies, but fixing it throughout the project will require significant refactoring, so until then file-wide exceptions for this rule look reasonable, but let me know otherwise.